### PR TITLE
Increase the range of assertion in Concurrency_test()

### PR DESCRIPTION
**Pull request by smitpatel**
@anpete

### CODE REVIEW

File: test/EFCore.SqlServer.FunctionalTests/DbContextPoolingTest.cs [hunk 1]
```diff
@@ -349,7 +349,7 @@ public void Provider_services_are_reset()
 
             await Task.WhenAll(tasks);
 
-            Assert.InRange(PooledContext.InstanceCount, 30, 50);
+            Assert.InRange(PooledContext.InstanceCount, 50, 500);
```
Comment (anpete) [inline test/EFCore.SqlServer.FunctionalTests/DbContextPoolingTest.cs]: Keep lower bound

Comment (smitpatel) [inline test/EFCore.SqlServer.FunctionalTests/DbContextPoolingTest.cs]: Done

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,12 @@ os:
   - linux
 #  - osx
 osx_image: xcode8.1
+addons:
+  apt:
+    sources:
```
Comment (mikeharder) [inline .travis.yml]: Why is the `sources` element needed?  I think just `packages\libunwind8` should be sufficient.

Comment (smitpatel) [inline .travis.yml]: Unable to find package with just that.

Comment (mikeharder) [inline .travis.yml]: Yeah, I found this issue: https://github.com/travis-ci/travis-ci/issues/4780.  CC: @natemcmaster

Comment (natemcmaster) [inline .travis.yml]: libunwind8 shouldn't be required for .NET Core 2.0 IIRC

Comment (natemcmaster) [inline .travis.yml]: Try adding this instead:
```yml
group: deprecated-2017Q2
```

Travis changed the trusty images today: https://blog.travis-ci.com/2017-06-21-trusty-updates-2017-Q2-launch

Comment (smitpatel) [inline .travis.yml]: Saw that. Is there a point of going backwards?
> We are planning to keep the previous generation of images around until September 1st, 2017.

Comment (mikeharder) [inline .travis.yml]: @natemcmaster: Docker deps still installs libunwind8 so I assume it's required: https://github.com/dotnet/dotnet-docker-nightly/blob/master/2.0/runtime-deps/stretch/Dockerfile#L16.  Also the CLI install scripts block if libunwind not installed: https://github.com/dotnet/cli/blob/release/2.0.0-preview2/scripts/obtain/dotnet-install.sh#L194.

Comment (natemcmaster) [inline .travis.yml]: You're write. Looks like these are still required on debian.

image

**Pull request merged**